docs/reference/api-docs.md

```diff
@@ -1,2073 +1,13 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-api-reference.html
-navigation_title: API Reference
+navigation_title: API Reference (moved)
```

Comment:
I had to preserve that page because it is referenced in the `docs-content` repo, doc generation fails if I remove it. I believe we can fix that in a follow up PR once this one is merged.